### PR TITLE
GPIO 35 Wakeup fixed

### PR DESCRIPTION
**Pull request by roberbike**
## Description
PWR EN must be activated even in deepsleep. Powering the board through the PWR EN usb port It is not necessary.

Tested with TTGO T_Display activating deep sleep with usb connected and usb disconnected.

## Tasks

- [ ] test quiescent current
- [ ] test main sensor enable/disable state

### CODE REVIEW

File: lib/gui-utils-tft/src/TFTUtils.cpp [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #include "TFTUtils.hpp"
 
+#include "driver/rtc_io.h"
+
```
Comment (hpsaturn) [inline lib/gui-utils-tft/src/TFTUtils.cpp:4]: please move the headers, to the main header of this class, to TFTUtils.hpp

File: platformio.ini [hunk 2]
```diff
@@ -17,7 +17,7 @@ framework = arduino
 upload_speed = 1500000
 monitor_speed = 115200
 version = 0.4.6
-revision = 838
+revision = 938
```
Comment (hpsaturn) [inline platformio.ini:20]: please use 843

**Review by hpsaturn — CHANGES_REQUESTED** [review]

File: lib/gui-utils-tft/src/TFTUtils.cpp [hunk 3]
```diff
@@ -431,6 +433,12 @@ void TFTUtils::suspend() {
     esp_bluedroid_disable();
     esp_bt_controller_disable();
     esp_wifi_stop();
+
+    rtc_gpio_init(GPIO_NUM_14);
+    rtc_gpio_set_direction(GPIO_NUM_14, RTC_GPIO_MODE_OUTPUT_ONLY);
+	rtc_gpio_set_level(GPIO_NUM_14, 1);
```
Comment (hpsaturn) [inline lib/gui-utils-tft/src/TFTUtils.cpp:439]: please change GPIO_NUM_14 to ADC_EN (hal.h define)

**Review by hpsaturn — CHANGES_REQUESTED** [review]

File: platformio.ini (hunk 2 above)
Comment (roberbike) [reply platformio.ini:20]: Done

File: lib/gui-utils-tft/src/TFTUtils.cpp (hunk 3 above)
Comment (roberbike) [reply lib/gui-utils-tft/src/TFTUtils.cpp:439]: Don´t admit this changue, only GPIO_NUM_14

File: lib/gui-utils-tft/src/TFTUtils.cpp (hunk 1 above)
Comment (roberbike) [reply lib/gui-utils-tft/src/TFTUtils.cpp:4]: Done

File: lib/gui-utils-tft/src/TFTUtils.cpp (hunk 3 above)
Comment (hpsaturn) [reply lib/gui-utils-tft/src/TFTUtils.cpp:437]: ok

**Pull request merged**